protocol/shannon/fullnode_cache.go

```diff
@@ -55,11 +45,12 @@ const (
 	evictionPercentage = 10
 
 	// minEarlyRefreshPercentage: Minimum percentage of the TTL before the cache early refresh may start.
-	// For example, if the TTL is 4 minutes, the cache will start refreshing at 1.2 minutes.
-	minEarlyRefreshPercentage = 0.3
+	// For a 30-second TTL, this means refresh can start at 22.5 seconds (75% of 30s).
```

Comment:
Please add a TODO for revisiting session fetch mechanism: we can use the block client in Shannon SDK to make informed decisions on when to fetch new sessions.